### PR TITLE
fix: unstable deployment databend

### PR DESCRIPTION
**Pull request by hanyisong**
Signed-off-by: Yisong Han <yisong8686@gmail.com>

related: https://github.com/sqlancer/sqlancer/issues/585#issuecomment-1275931403

### CODE REVIEW

**Comment by hanyisong** [comment]
cc @mrigger

**Comment by hanyisong** [comment]
When databend has not been compiled, other DBMSs have completed the ci test lol.

**Comment by hanyisong** [comment]
cc @Xuanwo

**Comment by mrigger** [comment]
Looks good to me. Thanks! @Xuanwo and @hanyisong, is this PR ready to be merged?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -134,10 +134,24 @@ jobs:
         run: |
           sudo apt update
           sudo apt install mysql-client
-          LASTEST_TAG=$(curl -s GET https://api.github.com/repos/datafuselabs/databend/tags\?per_page\=1 | jq -r '.[].name')
-          curl -LJO https://github.com/datafuselabs/databend/releases/download/${LASTEST_TAG}/databend-${LASTEST_TAG}-x86_64-unknown-linux-musl.tar.gz
-          mkdir ./databend && tar xzvf databend-${LASTEST_TAG}-x86_64-unknown-linux-musl.tar.gz -C ./databend
-          ./databend/bin/databend-query &
+          
+          # The first program: Download databend from github (unstable)
```
Comment (Xuanwo) [inline .github/workflows/main.yml:138]: Let's remove those not-used jobs to reduce the maintainer's burden.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -134,10 +134,24 @@ jobs:
...
+          # The second program: git clone and then compiled to install databend (The dependencies are not easy to solve, and the use time is more) 
+          # Dependecy protoc unresolved
+          # TODO: This method can be replaced with https://databend.rs/doc/contributing/building-from-source#build-with-docker-image
+          # git clone https://github.com/datafuselabs/databend.git
+          # cd databend && cargo build --bin databend-query --release
+          # ./target/release/databend-query &
+
+          # The third program: Download databend-query from docker hub
+          docker pull datafuselabs/databend-query
+          docker run --net=host datafuselabs/databend-query --mysql-handler-host 0.0.0.0 &
```
Comment (Xuanwo) [inline .github/workflows/main.yml:153]:
```suggestion
          docker run --net=host datafuselabs/databend-query
```

If we use `--net=host`, we don't need to specify the `mysql-handler-host` anymore.

**Comment by hanyisong** [comment]
cc @mrigger The maven dependency download is unstable. If re-executing the github action can pass the ci test, I think it can be merged.
https://github.com/sqlancer/sqlancer/actions/runs/3264422096/jobs/5365042249#step:4:461

[image](https://github.com/sqlancer/sqlancer/actions/runs/3264422096/jobs/5365042249#step:4:461)

**Comment by mrigger** [comment]
Sounds good. Thanks, @Xuanwo and @hanyisong!

**Pull request merged**